packages/adminrouter/extra/apache-windows/adminrouter.conf

```diff
@@ -0,0 +1,143 @@
+LoadModule actions_module modules/mod_actions.so
+LoadModule alias_module modules/mod_alias.so
+LoadModule allowmethods_module modules/mod_allowmethods.so
+LoadModule asis_module modules/mod_asis.so
+LoadModule auth_basic_module modules/mod_auth_basic.so
+LoadModule authn_core_module modules/mod_authn_core.so
+LoadModule authn_file_module modules/mod_authn_file.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule authz_groupfile_module modules/mod_authz_groupfile.so
+LoadModule authz_host_module modules/mod_authz_host.so
+LoadModule authz_user_module modules/mod_authz_user.so
+LoadModule autoindex_module modules/mod_autoindex.so
+LoadModule cgi_module modules/mod_cgi.so
+LoadModule dir_module modules/mod_dir.so
+LoadModule env_module modules/mod_env.so
+LoadModule include_module modules/mod_include.so
+LoadModule isapi_module modules/mod_isapi.so
+LoadModule log_config_module modules/mod_log_config.so
+LoadModule mime_module modules/mod_mime.so
+LoadModule negotiation_module modules/mod_negotiation.so
+LoadModule proxy_module modules/mod_proxy.so
+LoadModule proxy_http_module modules/mod_proxy_http.so
+LoadModule setenvif_module modules/mod_setenvif.so
+
+<IfModule unixd_module>
+User daemon
+Group daemon
+</IfModule>
+
+DocumentRoot "c:\opt\mesosphere\bin\Apache24\htdocs"
+
+<Directory />
+    AllowOverride none
+    Require all denied
+</Directory>
+
+# Configures the footer on server-generated documents
+ServerSignature Off
+
+# Configures to send just "Server:Apache" in the Server HTTP response header 
+ServerTokens Prod
+
+<VirtualHost *:61001>
+    # Enforce 1024M request body max size limit for DC/OS 
+    LimitRequestBody 1073741824
+    # Set keepalive connections timeout for DC/OS
+    KeepAliveTimeout 65
+    # Settings for Apache proxy options
+    ProxyPreserveHost On
+    # Disable forward proxy
+    ProxyRequests Off
+    <Proxy *>
+        Order allow,deny
+        Allow from all
+    </Proxy>
+
+    # Proxy routing for DC/OS Metrics service
+    <Location /system/v1/metrics>
+        ProxyPass        http://localhost:9000
+        ProxyPassReverse http://localhost:9000
+    </Location>
+
+    # Proxy routing for DC/OS Diagnostics service
+    <Location /system/health/v1>
+        ProxyPass        http://localhost:9003/system/health/v1
+        ProxyPassReverse http://localhost:9003/system/health/v1
+    </Location>
+    # Proxy routing for DC/OS Pkgpanda service
+    <Location /pkgpanda>
+        ProxyPass        http://localhost:9001
+        ProxyPassReverse http://localhost:9001 
+    </Location>
+    # Routing for DC/OS Logging service
+    <Location /system/v1/logs>
+        ProxyPass        http://localhost:9002
+        ProxyPassReverse http://localhost:9002
+    </Location>
+
+</VirtualHost>
+
+<IfModule dir_module>
+    DirectoryIndex index.html
+</IfModule>
+
+<Files ".ht*">
+    Require all denied
+</Files>
+
+ErrorLog "logs/error.log"
+LogLevel warn
+
+<IfModule log_config_module>
```

Comment:
I do not think this should be inside an `ifmodule` block as we always want logs to be present and will never disable logging/log_config module.

Reply:
will reorganize this.